### PR TITLE
Moves around fenced div introduction and exercises

### LINKED ISSUE #415: Move fenced divs to a separate subsection in adding lesson content episode
### How could the content be improved?

Fenced divs should be covered in a separate subsection in https://carpentries.github.io/lesson-development-training/lesson-content.html. They could either be introduced after lesson's home page before we go into adding them in episodes, or after adding episodes section. The more I think about this, the more I am inclined to cover them before adding episodes - e.g. have a section that talks about structural components of page content.

### Which part of the content does your suggestion apply to?

https://carpentries.github.io/lesson-development-training/lesson-content.html

### PR DESCRIPTION
**Pull request by anenadic**
Fixes #415.

### CODE REVIEW

File: episodes/lesson-content.md [hunk 1]
```diff
@@ -42,11 +51,64 @@ site: sandpaper::sandpaper_site
...
+::: callout 
+
+### Fenced divs
+
+Fenced divs are structural elements within the page, which get rendered in a visually distinct 
+way on the lesson website.
+Fenced divs include regular Markdown content surrounded by special tags to mark 
+the start and the end of such a block. 
+
+To mark the start of a fenced div, start a line with a start tag - at least 3 colon characters
```
[the remark this answers is not in the document]
Comment (tobyhodges) [reply episodes/lesson-content.md:85]: Trying to avoid repeated use of "start"

File: episodes/lesson-content.md [hunk 2]
```diff
@@ -42,11 +51,64 @@ site: sandpaper::sandpaper_site
...
+Fenced divs are structural elements within the page, which get rendered in a visually distinct 
+way on the lesson website.
+Fenced divs include regular Markdown content surrounded by special tags to mark 
+the start and the end of such a block. 
+
+To mark the start of a fenced div, start a line with a start tag - at least 3 colon characters 
+followed by a blank character and a keyword denoting the type of a fenced div you are 
+creating (`::: fenced_div_keyword`) . 
+Then you add the Markdown content of this structural block. 
+Finally, end your fenced div with an end tag consisting of at least 3 colons (`:::`).
```
Comment (tobyhodges) [inline episodes/lesson-content.md:89]:
```suggestion
Finally, end your fenced div with a closing tag consisting of at least 3 colons (`:::`).
```

File: episodes/lesson-content.md [hunk 3]
```diff
@@ -77,59 +139,36 @@ of how content will be displayed in your website**.
...
-   and we will explore some of them in this episode.
+Like `index.md`, the episode file begins with a short header.
+The fields in this header describe the episode title and the estimated time (in minutes)
+required to teach it and complete the exercises.
 
 ### Creating a new episode
 
-Let's create a new episode file, for one of the episodes you have just identified.
+Let's create a new episode file, for one of the episodes you have identified earlier in the 
+training.
```
Comment (tobyhodges) [inline episodes/lesson-content.md:160]:
```suggestion
Let's create a new episode file, for one of the episodes you have identified earlier in the training.
```

File: episodes/lesson-content.md [hunk 4]
```diff
@@ -77,59 +139,36 @@ of how content will be displayed in your website**.
...
 First, open the "raw" view of the `introduction.md` example episode,
-and copy its contents.
+and copy its contents to make it easier to structure the new episode we are creating.
+
+::: instructor
+
+You may ask each of the member of a the lesson team to create an episode file for
```
Comment (tobyhodges) [inline episodes/lesson-content.md:166]:
```suggestion
You may ask each trainee in a lesson team to create an episode file for
```

File: episodes/lesson-content.md [hunk 5]
```diff
@@ -188,7 +224,7 @@ It is vital to include the file extension when naming this file:
 only files with the `.md` or `.Rmd` extensions will be built into
 webpages by the lesson infrastructure.
 
-For page content, paste the content the `introduction.md` file you copied earlier and:
+For the content of your episode, paste the content the `introduction.md` file you copied earlier and:
```
Comment (tobyhodges) [inline episodes/lesson-content.md:227]:
```suggestion
For the content of your episode, paste everything from the `introduction.md` file you copied earlier and:
```

File: episodes/lesson-content.md [hunk 6]
```diff
@@ -301,15 +292,15 @@ Using this approach, we can build up our lesson one episode at a time.
 ## Adding Exercises
 
 To create an exercise in The Carpentries Workbench,
-we use another two types of 'fenced divs':
+we use another two types of fenced divs:
```
Comment (tobyhodges) [inline episodes/lesson-content.md:295]:
```suggestion
we use another two types of fenced div:
```

File: episodes/lesson-content.md [hunk 7]
```diff
@@ -301,15 +292,15 @@ Using this approach, we can build up our lesson one episode at a time.
...
 
 * `discussions`, where the main task is for participants to discuss a topic or prompt, and 
 * `challenges`, where the main task is a problem to be solved.
 
-For example, to start a challenge fenced div, use at least 3 colons, 
-followed by the `challenge` tag.
-Then the content of the challenge is included on the following lines.
-Finally, you need to close the fenced div using another line with least 3 colons.
+To start a challenge fenced div, as usual use at least 3 colons, then followed it by the 
+`challenge` keyword.
```
Comment (tobyhodges) [inline episodes/lesson-content.md:301]:
```suggestion
To start a challenge fenced div, write an opening tag of at least 3 colons as usual, then follow it with the `challenge` keyword.
```

File: episodes/lesson-content.md [hunk 8]
```diff
@@ -419,7 +413,7 @@ Use this time to add more content to your lesson site.
 Generally speaking, you should try to transfer the drafted content from your lesson design notes into the lesson website.
 Here are some suggestions for things you might try:
 
-1. Add your list of prerequisite knowledge/skills as a fenced div with the `prereq` class to your `index.md` file.
+1. Add different types of exercises - e.g. add a `discussion`.
```
Comment (tobyhodges) [inline episodes/lesson-content.md:416]: I like this suggestion, but do we need to add a note about `solution`s not working well with `discussion`s?

**Review by tobyhodges — CHANGES_REQUESTED** [review]
Thanks @anenadic, especially for the annotated screenshots, which are a huge improvement. I have suggested a few language polishes, and asked a question about how we want to frame `discussion`s as different from `challenge`s

File: episodes/lesson-content.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (anenadic) [reply episodes/lesson-content.md:85]: Indeed :-) - begin, start and openning to the rescue.

File: episodes/lesson-content.md (hunk 8 above)
Comment (anenadic) [reply episodes/lesson-content.md:414]: Is this the issue where "solutions" do not get rendered within "discussions" @tobyhodges ? I forgot ehat the issue was but remember that something was wrong there.

Comment (tobyhodges) [reply episodes/lesson-content.md:414]: That's right: the Workbench assumes that discussions do not have a solution.

Comment (tobyhodges) [reply episodes/lesson-content.md:414]:
```suggestion
1. Add other exercises - e.g. add a `discussion`. Note that the lesson infrastructure does not support `solution` divs to be attached to `discussion`s.
```

**Review by tobyhodges — APPROVED** [review]
Thanks @anenadic 🙌

**Comment by anenadic** [comment]
Oh, you have done it already - many thanks @tobyhodges 🎉 .

**Pull request merged**